scripts/setup.mongodb

```diff
@@ -55,9 +55,9 @@ try {
     console.log(`Populating system types in ${databaseName}...`);
 
     db.system_types.insertMany([
-      { _id: SYSTEM_TYPE_ID_STORAGE, value: "Storage" },
-      { _id: SYSTEM_TYPE_ID_OPERATIONAL, value: "Operational" },
-      { _id: SYSTEM_TYPE_ID_SCRAPPED, value: "Scrapped" },
+      { _id: SYSTEM_TYPE_ID_STORAGE, value: "Storage", description: "storage system type" },
+      { _id: SYSTEM_TYPE_ID_OPERATIONAL, value: "Operational", description: "operational system type" },
+      { _id: SYSTEM_TYPE_ID_SCRAPPED, value: "Scrapped", description: "scrapped system type" },
```

Comment:
```suggestion
      { _id: SYSTEM_TYPE_ID_STORAGE, value: "Storage", description: "Storage system type" },
      { _id: SYSTEM_TYPE_ID_OPERATIONAL, value: "Operational", description: "Operational system type" },
      { _id: SYSTEM_TYPE_ID_SCRAPPED, value: "Scrapped", description: "Scrapped system type" },
```
Although I dont think what we use really matters right now anyway, I spoke with Joshua and they could be more like "Contains items that are in storage", or "An operational system, items inside are classified as in use" or something perhaps as a hover tooltip on the frontend. It will probably be clearer down the line when the frotend issue is done. We also don't know what CLF would want to use.